app/server/app.ts

```diff
@@ -87,7 +87,14 @@ export async function setupApp(
   app.use((req, res, next) => {
     const connect = req.cookies['connect.sid'];
 
-    res.cookie('connect.sid', connect, { secure: true, sameSite: 'strict' });
+    if (isFeatureEnabled(Feature.HTTPONLY_COOKIE_FLAG_ENABLED)) {
+      res.cookie('connect.sid', connect, {
+        secure: true,
+        httpOnly: true,
+        sameSite: true,
+      });
+    }
```

Comment:
I think the feature flag should be for the values, not for setting up the cookie itself. because we still want to setup the cookie in production, we just don't want to set certain flags to true, so we can make the value conditional based on whether the flag is turned on or off